docs/add-a-system-config.rst

```diff
@@ -43,15 +44,23 @@ in :doc:`system-list`.
 If you are running on a system with an accelerator, find an existing system with the
 same accelerator vendor, and then secondarily, if you can, match the actual accelerator.
 
-1. accelerator.vendor
-2. accelerator.name
+1. ``accelerator.vendor`` - Company name
```

Comment:
Do we think the hardware description deserves multiple sections? Or is it pretty self-explanatory from the existing examples (I think the latter).